--- qt-everywhere-src-5.13.0/qtbase/mkspecs/features/qmake_use.prf	2019-06-12 22:18:18.000000000 -0600
+++ qt-everywhere-src-5.13.0.mod/qtbase/mkspecs/features/qmake_use.prf	2020-10-15 13:31:21.826629229 -0600
@@ -22,6 +22,8 @@
         !defined(QMAKE_LIBS_$$nu, var): \
             error("Library '$$lower($$replace(nu, _, -))' is not defined.")
 
+        QMAKE_LIBDIR += $$eval(QMAKE_LIBDIR_$$nu)
+
         debug: \
             LIBS$${suffix} += $$eval(QMAKE_LIBS_$${nu}_DEBUG) $$eval(QMAKE_LIBS_$$nu)
         else: \
